src/pages/common/AuthFormComponent/LoginFormComponent.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { useAppDispatch } from "../../../hooks/useAppDispatch";
+import { Formik, Form, Field } from "formik";
+import "./style.css";
+import * as yup from "yup";
+import { login } from "../../../store/reducers/actionCreator";
+import { useAppSelector } from "../../../hooks/useAppSelector";
+import Loader from "../../../components/Loader/Loader";
+
+const LoginFormComponent = () => {
+  const dispatch = useAppDispatch();
+  const { loading, errorLogin } = useAppSelector((state) => state.authReducer);
+
+  const Login = (email: string, password: string) => {
```

Comment:
Use lowercase. Uppercase is for Classes, Components, Enums and some const values